decidim-templates/spec/system/admin/admin_manages_proposal_answer_templates_spec.rb

```diff
@@ -0,0 +1,217 @@
+          :proposal_answer_template_name,
+          "#proposal_answer_template-name-tabs",
+          en: "My new name",
+          es: "Mi nuevo nombre",
+          ca: "El meu nou nom"
+        )
+
+        select scope_name, from: :proposal_answer_template_scope_for_availability
+
+        within ".edit_proposal_answer_template" do
+          page.find("*[type=submit]").click
+        end
+
+        expect(page).to have_admin_callout("successfully")
+        expect(page).to have_current_path decidim_admin_templates.proposal_answer_templates_path
+        within ".proposal_answer-templates" do
+          expect(page).to have_i18n_content(scope_name)
+          expect(page).to have_content("My new name")
+        end
+      end
+    end
+
+    it_behaves_like "updates a template with scopes", "Global (available everywhere)"
+    it_behaves_like "updates a template with scopes", "Participatory process: A participatory process > A component"
+  end
+
+  describe "updating a template with invalid values" do
+    let!(:template) { create(:template, :proposal_answer, organization:) }
+
+    before do
+      visit decidim_admin_templates.proposal_answer_templates_path
+      click_link translated(template.name)
+    end
+
+    it "does not update the template" do
+      fill_in_i18n(
+        :proposal_answer_template_name,
+        "#proposal_answer_template-name-tabs",
+        en: "",
+        es: "",
+        ca: ""
+      )
+
+      within ".edit_proposal_answer_template" do
+        find("*[type=submit]").click
+      end
+
+      expect(page).to have_admin_callout("problem")
+    end
+  end
+
+  describe "copying a template" do
+    let!(:template) { create(:template, :proposal_answer, organization:) }
+
+    before do
+      visit decidim_admin_templates.proposal_answer_templates_path
+    end
+
+    it "copies the template" do
+      within find("tr", text: translated(template.name)) do
+        click_link "Duplicate"
+      end
+
+      expect(page).to have_admin_callout("successfully")
+      expect(page).to have_content(template.name["en"], count: 2)
+    end
+  end
+
+  describe "destroying a template" do
+    let!(:template) { create(:template, :proposal_answer, organization:) }
+
+    before do
+      visit decidim_admin_templates.proposal_answer_templates_path
+    end
+
+    it "destroys the template" do
+      within find("tr", text: translated(template.name)) do
+        accept_confirm { click_link "Delete" }
+      end
+
+      expect(page).to have_admin_callout("successfully")
+      expect(page).to have_no_i18n_content(template.name)
+    end
+  end
+
+  describe "using a proposal_answer_template" do
+    let(:participatory_process) { create(:participatory_process, title: { en: "A participatory process" }, organization:) }
+    let!(:component) { create(:component, manifest_name: :proposals, name: { en: "A component" }, participatory_space: participatory_process) }
+
+    let(:description) { "Some meaningful answer" }
```

Comment:
Can you add some examples about the string interpolation feature? (`%{organization}`, `%{admin}` , `%{user}`)

Reply:
done in de18fa124e